mscore/events.cpp

```diff
@@ -399,6 +399,11 @@ void ScoreView::mousePressEventNormal(QMouseEvent* ev)
                   }
             else {
                   if (st == SelectType::ADD) {
+                        // convert range to list
+                        if (e->score()->selection().isRange()) {
+                              e->score()->selection().setState(SelState::LIST);
```

Comment:
This has an effect on all operations acting differently on the selection being a list or a range, e.g. <code>Palette::applyPaletteElement()</code>.
Selecting a range and double click on a double barline, the barline at the **end** of the measure is changed. With this PR, a double barline is created just **before** all selected <code>ChordRest</code>'s.
If only one <code>ChordRest</code> is selected, the double barline is create again at the end of the measure.

Reply:
Sorry, I'm not following.  This function isn't called when applying palette elements, only when actually clicking within the score.  And the change here only applies to Ctrl+click.  Unless there is something I'm missing?  Were you able to duplicate a problem?  I am not able to, but maybe I am misunderstand the steps.

Reply:
Hm, I can't reproduce it myself now. When I tried yesterday I got a very weird situation. I think it was related of a list which was, using this patch, converted into a range. And the problem is <code>Palette::applyPaletteElement()</code> handles the dropped element differently whether the selection is a list or a range.
Sorry, most likely not an issue at all, just confused with the different behavior of lists and ranges by <code>Palette::applyPaletteElement()</code>.